docs/source/serialization.rst

```diff
@@ -430,3 +430,67 @@ Using the traced model for inference is as simple as using its ``__call__`` dund
 .. code-block:: python
 
     traced_model(tokens_tensor, segments_tensors)
+
+Using HuggingFace TorchScript model in AWS Inf1 using Neuron SDK
+-----------------------------------------------------------------------------------------------------------------------
+
+AWS introduced `Inf1 <https://aws.amazon.com/ec2/instance-types/inf1/>`_ for low cost, high performance machine learning model inference tasks in AWS cloud. 
```

Comment:
We should think about calling inferentia: `inf1` or `inferentia` in the whole document and not mix them up. This can confuse none aws experienced people

Reply:
Agreed and all should be Inf1. Changes made in new commit.

Reply:
Appreciate the suggestion and changes are made where appropriate. We used the term 'Inf1' to refer to EC2 instance type, and 'Inferentia' to refer to the chip name. Rebased and resubmit in a new PR: https://github.com/huggingface/transformers/pull/14982